.github/workflows/build.yml

```diff
@@ -51,22 +52,23 @@ jobs:
         uses: actions/setup-node@v2.1.5
         with:
           node-version: 14
+          check-latest: true
```

Comment:
From the docs:

> Setting check-latest to true has performance implications as downloading versions of Node is slower than using cached versions

They also mention that for maximum stability, it should be left to the default.

It's likely best to keep this to the default, since we don't need the absolute latest version of Node to run these. I'd rather have the workflows run as fast as possible so we don't have to wait too much when rebasing or something.